Gruntfile.js

```diff
@@ -31,6 +31,9 @@ module.exports = function(grunt) {
           'lib/iced-coffee-script.js':
              'iced-coffee-script/extras/iced-coffee-script-1.8.0-c.js',
           'lib/jquery.js' : 'jquery/dist/jquery.js',
+          'lib/jquery-ui.min.js' : 'jquery-ui/jquery-ui.min.js', 
+          'lib/jquery-ui-slider-pips.min.js': 'jquery-ui-slider-pips/dist/jquery-ui-slider-pips.min.js',
+          'lib/jquery-ui-slider-pips.css': 'jquery-ui-slider-pips/dist/jquery-ui-slider-pips.css', 
```

Comment:
All the images that you have also copied should be included here, if they are needed.  (If they are not needed, then we don't need to check in those files.)